tests/test.py

```diff
@@ -4785,6 +4785,128 @@ def testEmptyValues(self):
         self.assertEqual(row['binary'], b'')
 
 
+class TestEncryptionWithIntFields(BaseTestCase):
```

Comment:
hm, if we just check that field left as is, we can just add it to `TestTransparentEncryption`/`TestTransparentEncryptionWithZone` tests and extend table. There we check that all is correct when result contains null values, empty strings and we can add integer too as special case

@ilammy what you think about it? do we need separate test case which do the same and take a time for execution. or we can all such cases merge into one testcase?

Reply:
> do we need separate test case which do the same and take a time for execution. or we can all such cases merge into one testcase?

Mmm... Part of me thinks that it's a good thing to have separate tests where each of them tests their own thing, as that makes it easier to pinpoint the cause if one of them breaks. However, considering that we already have a test for transparent encryption which basically tests all item types, I guess it also makes sense to just extend it instead of adding a new one.

So I'm for the merge. @iamnotacake, please do :pray: